front/src/components/molecules/FriendInfoContainer/FriendInfoContainer.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { styled } from '@material-ui/core/styles';
+import Avatar from '../../atoms/Avatar/Avatar';
+import Typo from '../../atoms/Typo/Typo';
+import IconButton from '../../atoms/IconButton/IconButton';
+import { FriendInfoContainerProps } from '../../../interfaces/molecules/FriendInfoContainer/IFriendInfoContainer';
+
+const RootContainer = styled(Grid)({
+  width: '100%',
+  justifyContent: 'center',
+  alignItems: 'center',
+  backgroundColor: '#f5f6fa',
+  borderRadius: '10px',
+});
+
+const ItemContainer = styled(Grid)({
+  position: 'relative',
+  paddingLeft: 10,
+});
+
+const NameGrid = styled(Grid)({
+  '&:hover ~ div': {
+    position: 'absolute',
+    display: 'block',
+    bottom: '70%',
+    left: '70%',
+  },
+});
+
+const OnlineGrid = styled(Grid)({
+  color: '#50CB93',
+  '&:hover ~ div': {
+    position: 'absolute',
+    display: 'block',
+    bottom: '70%',
+    left: '70%',
+  },
+});
+
+const OfflineGrid = styled(Grid)({
+  color: '#B2B1B9',
+  '&:hover ~ div': {
+    position: 'absolute',
+    display: 'block',
+    bottom: '70%',
+    left: '70%',
+  },
+});
+
+const RemoveFriendGrid = styled(Grid)({
+  '&:hover ~ div': {
+    position: 'absolute',
+    display: 'block',
+    bottom: '60%',
+    left: '40%',
+  },
+});
+
+const InfoGrid = styled(Grid)({
+  display: 'none',
+  justifyContent: 'center',
+  fontSize: '14px',
+  borderRadius: '3px',
+  padding: '2px 6px',
+  backgroundColor: '#a4b0be',
+  color: 'white',
+  '& p': {
+    margin: 0,
+  },
+});
+
+const FriendInfoContainer = ({
+  src, name, isOnline, showUserProfile, removeFriend,
+}: FriendInfoContainerProps): React.ReactElement => {
+  const onlineState = isOnline ? 'online' : 'offline';
+
+  return (
+    <RootContainer container onClick={showUserProfile}>
+      <ItemContainer item>
+        <Avatar src={src} size="large" />
+      </ItemContainer>
+      <ItemContainer item xs={10}>
+        <Grid container justifyContent="space-evenly" alignItems="center">
+          <ItemContainer item xs={3}>
+            <Grid container direction="column" alignItems="center">
+              <NameGrid>
+                <Typo variant="h6">{name}</Typo>
+              </NameGrid>
+              <InfoGrid><p>이름</p></InfoGrid>
+            </Grid>
+          </ItemContainer>
+          <ItemContainer item xs={3}>
+            <Grid container direction="column" alignItems="center">
+              {
+                isOnline ? (<OnlineGrid><Typo variant="h6">{onlineState}</Typo></OnlineGrid>)
+                  : (<OfflineGrid><Typo variant="h6">{onlineState}</Typo></OfflineGrid>)
```

Comment:
```suggestion
                isOnline ? <OnlineGrid /> : <OfflineGrid />
```

OnlineGrid, OfflineGrid의 컴포넌트 차일드에 Typo도 같이 넣어서 정의했으면 좋을 것 같습니다.
현재 코드에서 결국 onlineState라는 값 자체도 isOnline에 대해 정의됨으로 이를 중복 언급하는 것을 피하면 더 좋을 것 같습니다.